### PR TITLE
[MM-57460] - Calls: Host controls, add mute all to participant list

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Add the mute all command in the participant list for hosts
- Designs: https://www.figma.com/file/8C0ojMc778Zx3a0nm66Edw/MM-56584-Host-Controls?node-id=1270%3A54648&mode=dev

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-57470

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] Includes text changes and localization file updates
- [x] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
- Android: 13, Pixel 6
- Android: 13, Galaxy Tab s7+
- iOS: 16.5.1, iPhone 14

#### Release Note
```release-note
Calls: Host controls, mute all option added to participants list
```

### CODE REVIEW

File: app/products/calls/screens/participants_list/participants_list.tsx [hunk 1]
```diff
@@ -63,6 +84,14 @@ export const ParticipantsList = ({closeButtonId, sessionsDict, teammateNameDispl
     if (sessions.length > MIN_ROWS && snapPoint1 < snapPoint2) {
         snapPoints.push(snapPoint2);
     }
+    const unMuted = useMemo(() => sessions.some((s) => !s.muted), [sessions]);
+
+    const muteAllPress = useCallback(async () => {
+        if (!callServerUrl || !callChannelId) {
+            return;
+        }
+        hostMuteAll(callServerUrl, callChannelId);
```
[the remark this answers is not in the document]
Comment (cpoile) [reply app/products/calls/screens/participants_list/participants_list.tsx:93]: Ah, good catch, thank you. Fixed.

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
This looks good to me @cpoile. The only thing I'm thinking is that `Mute all` should not mute the host who has clicked mute all. I think the main use case is to mute everyone except yourself. Thoughts?

Other than that, I think this is working well

**Review by streamer45 — APPROVED** [review]
Looks good, pending label changes.

**Comment by cpoile** [comment]
@matthewbirtch Updated label, logic is same as https://github.com/mattermost/mattermost-plugin-calls/pull/726, screenshot in summary updated

**Review by matthewbirtch — APPROVED** [review]

**Pull request merged**